bundles/io/org.eclipse.smarthome.io.net/META-INF/MANIFEST.MF

```diff
@@ -10,6 +10,7 @@ Export-Package:
  org.eclipse.smarthome.io.net.exec,
  org.eclipse.smarthome.io.net.http
 Import-Package: 
+ javax.servlet.http;version="3.1.0",

```

Comment:
Which feature of the Servlet are we using that has been introduced in 3.1.0 and is not available before?

Reply:
none, the version was added by the IDE quick fix action, removed.